README.md

```diff
@@ -28,7 +28,7 @@ A bundle will be produced from the native packager settings of this project. A b
 packager distribution and includes some component configuration. To load the bundle:
 
 ```bash
-loadBundle <HIT THE TAB KEY AND THEN RETURN>
+conductr:loadBundle <HIT THE TAB KEY AND THEN RETURN>
```

Comment:
If the goal is discoverability, then do you agree that a `conduct` command achieves this goal given tab completion?

I'm trying to understand why having a scope would be better than a command, given that the latter more closely resembles our Python CLI.